WordPress/src/main/java/org/wordpress/android/ui/posts/prepublishing/home/usecases/GetPublishDateLabelUseCase.kt

```diff
@@ -0,0 +1,29 @@
+package org.wordpress.android.ui.posts.prepublishing.home.usecases
+
+import org.wordpress.android.R
+import org.wordpress.android.fluxc.model.post.PostStatus.PRIVATE
+import org.wordpress.android.ui.posts.EditPostRepository
+import org.wordpress.android.ui.posts.PostSettingsUtils
+import org.wordpress.android.ui.utils.UiString
+import org.wordpress.android.ui.utils.UiString.UiStringRes
+import org.wordpress.android.ui.utils.UiString.UiStringText
+import javax.inject.Inject
+
+class GetPublishDateLabelUseCase @Inject constructor(private val postSettingsUtils: PostSettingsUtils) {
```

Comment:
Yes, this is what we should be doing. I am just realizing that the contents in this use case need to shared in the `PostSettingsUtils` so that the `PostSettings` screen can behave similarly to this bottom sheet. Thank you so much for pointing this out.

Reply:
I deleted this class and left the logic within the `PostSettingsUtils` untouched because it worked well.